.github/workflows/ci.yml

```diff
@@ -62,43 +62,69 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        container-runtime: [focal, buster-slim]
-        runtime: [linux-x64]
+        container-runtime: [focal, buster-slim, alpine]
     runs-on: ubuntu-latest
     name: ci/github/build-${{ matrix.container-runtime }}
     steps:
     - name: Checkout
       uses: actions/checkout@v2
-    - shell: bash
-      run: |
-        git fetch --prune --unshallow
+      with:
+        fetch-depth: 0
     - name: Install net5.0
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 5.0.x
+    - name: Setup QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
     - name: Calculate Version
       run: |
         dotnet tool restore
         version=$(dotnet tool run minver -- --tag-prefix=oss-v)-${{ matrix.container-runtime }}
         echo "VERSION=${version}" >> $GITHUB_ENV
+    - name: Setup Variables
+      id: variables
+      uses: kanga333/variable-mapper@master
```

Comment:
@thefringeninja do we want to pin a specific version here so that our CI doesn't break if there are any breaking changes?

Reply:
as discussed offline with @thefringeninja, variable-mapper doesn't seem to support the standard github action convention (v1,v2, etc.) and any breaking changes would be noticed quickly within a few hours so it should be fine.